src/browser/preload_scripts.ts

```diff
@@ -136,37 +152,66 @@ function fetchToCache(identity: Identity, preloadScript: PreloadInstance): Promi
     });
 }
 
+// resolves to type `PreloadFetched` on success
+// resolves to `undefined` when fetch fails to download asset to Chromium cache
+function fetchPlugin(identity: Identity, preloadPlugin: PreloadPlugin): Promise<FetchResponse> {
+    return new Promise((resolve, reject) => {
+        const sourceUrl = getConfigUrlByUuid(identity.uuid);
+        const msg: PluginQuery = {
+            topic: 'application',
+            messageId: '1',
+            action: 'query-plugin',
+            name: preloadPlugin.name,
+            version: preloadPlugin.version,
+            optional: preloadPlugin.optional,
+            sourceUrl
+        };
+
+        rvmMessageBus.publish(msg, (resp) => {
+            if (resp.payload.hasOwnProperty('path') && resp.payload.action === 'query-plugin') {
+                const scriptPath = `${resp.payload.path}\\${resp.payload.target}`;
```

Comment:
do we need to check that the `resp.payload.path` does not already end in a `\`? In general we should probably use the node `path` utility for stuff like this